templates/expo-template-bare-minimum/ios/HelloWorld/AppDelegate.h

```diff
@@ -3,6 +3,7 @@
 #import <UIKit/UIKit.h>
 
 #import <Expo/Expo.h>
+#import <ExpoModulesCore/EXAppDelegateWrapper.h>
```

Comment:
Actually my intention was to include this header in  `<Expo/Expo.h>` 🤔 I think it would be more clear for people, `ExpoModulesCore` might be a little bit confusing if people don't know about `expo-modules-core`